tests/kryoptic-init.sh

```diff
@@ -51,4 +51,13 @@ export TOKENCONFIGVARS="export KRYOPTIC_CONF=$TOKDIR/kryoptic.conf"
 export TESTPORT="34000"
 
 export SUPPORT_ALLOWED_MECHANISMS=1
-export SUPPORT_ML_DSA=1
+
+# Set SUPPORT_ML_DSA only if is not set already
+if [ -z "$SUPPORT_ML_DSA" ]; then
+
+    # ML-DSA support is available since version 1.2
+    SUPPORT_ML_DSA=0
+    if rpm -q kryoptic &>/dev/null && [[ "$(rpm -q --qf '%{V}' kryoptic)" =~ 1\.[01]\. ]]; then
```

Comment:
Sorry I was not clear, but I do not want this check in this file, as it will cause unnecessary errors on non RPM systems.

This file should only support the ability to override the  SUPPORT_ML_DSA from an external job and default to 1 otherwise.

The CI jobs can then test kryoptic's version based on whatever packaging system the build is running on and set the override there.